core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -499,6 +506,48 @@ angular.module('oppia').component('explorationEditorPage', {
             }
           )
         );
+        ctrl.directiveSubscriptions.add(
+          ConnectionService.onInternetStateChange.subscribe(
+            currentState => {
+              ctrl.hasNetworkConnection = currentState.hasNetworkConnection;
+              ctrl.hasInternetAccess = currentState.hasInternetAccess;
+              if (ctrl.hasNetworkConnection && ctrl.hasInternetAccess) {
+                if (ctrl.status === 'OFFLINE') {
```

Comment:
Please use a named constant for strings like this.

Reply:
Done.